roofit/roofitcore/inc/RooSetProxy.h

```diff
@@ -13,64 +13,266 @@
  * with or without modification, are permitted according to the terms        *
  * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
  *****************************************************************************/
+
+/**
+\class RooSetCollectionProxy
```

Comment:
```diff
- \class RooSetCollectionProxy
+ \class RooCollectionProxy
```

Reply:
If you move it directly before the class declaration, the `\class` tag becomes unnecessary. We use it normally in the `.cxx` to keep the headers shorter, but here it's header only.

Reply:
Okay! I updated this.